flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -486,4 +630,14 @@ enum FlxAxes
 {
 	X;
 	Y;
+}
+
+enum GamepadModel
+{
+	Logitech;
+	OUYA;
+	PS3;
+	PS4;
+	Xbox;
```

Comment:
Shouldn't this be Xbox360 instead? Or rather, is Xbox360 identical to other xbox controllers?